pyuvdata/uvdata/tests/test_mir.py

```diff
@@ -347,8 +317,8 @@ def test_read_mir_write_ms_flex_pol(mir_data, tmp_path):
 
     # Read in the raw data so that we can manipulate it, and make it look like the
     # test data set was recorded with split-tuning
-    mir_data.sp_data["gunnLO"][np.isin(mir_data.sp_data["blhid"], [1, 3])] += 30.0
-    mir_data.sp_data["gunnLO"][np.isin(mir_data.sp_data["fsky"], [1, 3])] += 30.0
+    mir_data.sp_data._data["gunnLO"][np.isin(mir_data.sp_data["blhid"], [1, 3])] += 30.0
+    mir_data.sp_data._data["fsky"][np.isin(mir_data.sp_data["blhid"], [1, 3])] += 30.0
 
     # Spin up a Mir object, which can be covered into a UVData object,
     # with flex-pol enabled.
```

Comment:
It's true the tests are very similar, in part because they are leveraging similar pathways. I do think it's more clear to have those spelled out though, so I will take that argument that you have so conveniently offered ;) (although if I build any more tests with redundancy to this in the future, I might rethink the approach a little bit).